OneCommunityGlobal/HGNRest
# Luis edit warning description backend

# Description
This the backend which allows owners to edit warning descriptions. 
Owners should be allowed to edit warning descriptions and only owners

## Related PRS (if any):
To test this backend PR you need to checkout the frontend PR #2077

## Main changes explained:
- i added a new controller function `editWarningDescription.js` which handles logic to updating and setting the new value on the backend. 
- It returns a message confirming the description was edited successfully 

## How to test:
1. check into current branch
2. do `npm install` and `npm run build && npm run start` to run this PR locally
3. Follow the instructions of the Frontend PR
